docs/options.md

```diff
@@ -324,6 +324,115 @@ a set of operations as independent compilation passes.
 Note: This option may be removed in future Babel versions as we add better
 support for defining ordering between plugins.
 
+## Output targets
+
+### `targets`
+
+Type: `string | Array<string> | { [string]: string }`<br />
+Default: `{}`<br />
+Placement: Allowed in Babel's programmatic options, or in config files<br />
+Added in: `v7.13.0`<br />
+
+Describes the environments you support/target for your project.
+
+This can either be a [browserslist-compatible](https://github.com/ai/browserslist) query (with [caveats](#ineffective-browserslist-queries)):
+
+```json
+{
+  "targets": "> 0.25%, not dead"
+}
+```
+
+Or an object of minimum environment versions to support:
+
+```json
+{
+  "targets": {
+    "chrome": "58",
+    "ie": "11"
+  }
+}
+```
+
+Example environments: `chrome`, `opera`, `edge`, `firefox`, `safari`, `ie`, `ios`, `android`, `node`, `electron`.
+
+#### No targets
+
+When no targets are specified: Babel will assume you are targeting the oldest browsers possible. For example, `@babel/preset-env` will transform all ES2015-ES2020 code to be ES5 compatible.
+
+> We recommend setting `targets` to reduce the output code size.
+
+```json
+{
+  "presets": ["@babel/preset-env"]
+}
+```
+
+Because of this, Babel's behavior is different than [browserslist](https://github.com/browserslist/browserslist#queries): it does _not_ use the `defaults` query when there are no targets are found in your Babel _or_ browserslist config(s). If you want to use the `defaults` query, you will need to explicitly pass it as a target:
+
+```json
+{
+  "targets": "defaults"
+}
+```
+
+We recognize this isn’t ideal and will be revisiting this in Babel v8.
+
+#### `targets.esmodules`
+
+Type: `boolean`
+
+You may also target browsers supporting ES Modules (https://www.ecma-international.org/ecma-262/6.0/#sec-modules). When specifying this option, the browsers field will be ignored. You can use this approach in combination with `<script type="module"></script>` to conditionally serve smaller scripts to users (https://jakearchibald.com/2017/es-modules-in-browsers/#nomodule-for-backwards-compatibility).
+
+> _Please note_: when specifying both `browsers` and the esmodules target, they will be intersected.
+
+```json
+{
+  "targets": {
+    "esmodules": true
+  }
+}
+```
```

Comment:
I believe we should warn users who have set both `esmodules` and `browsers` options in their config.

Reply:
This is now well supported, since Babel will intersect them